clang/test/3C/root_cause.c

```diff
@@ -41,3 +41,13 @@ void test1() {
 extern int *glob; // expected-warning {{External global variable glob has no definition}}
 
 void (*f)(void *); // expected-warning {{Default void* type}}
+
+typedef struct {
+  int x;
+  float f;
+} A, *PA; // expected-warning {{Unable to rewrite a typedef with multiple names}}
+
+
+
+
+
```

Comment:
```suggestion
```
Remove blank lines.  (Even though I entered a suggestion to replace all 5 lines with the empty string, GitHub's preview shows the suggestion as removing only one line.  If you're curious, you could see what happens if you try to apply the suggestion.)